loginCompose/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same question

Reply:
removed it, was unsure if they belonged in each module or not